math/minuit/src/TFitter.cxx

```diff
@@ -137,8 +137,12 @@ void TFitter::GetConfidenceIntervals(Int_t n, Int_t ndim, const Double_t *x, Dou
    Double_t c=0;
 
    Double_t *matr = GetCovarianceMatrix();
-   if (!matr)
+   if (!matr){
+      delete [] grad;
```

Comment:
An example for this would be when you have a complex destructor (like in this case) and somebody decides to access or delete grad again. Assigning a nullptr will avoid future bugs from this kind.